user/inc/Arduino.h

```diff
@@ -91,7 +91,7 @@ inline void yield() {
 #endif
 
 #ifndef analogInputToDigitalPin
-#if PLATFORM_ID == PLATFORM_P2
+#if HAL_PLATFORM_RTL872X
```

Comment:
Please make changes to `portOutputRegister(port)`, `portInputRegister(port)` and `portModeRegister(port)` by replacing `(port == PORT_AB[RTL_PORT_A]) ? RTL_PORT_A : RTL_PORT_B` with `0`